health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/hooks/useProcessData.js

```
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import * as XLSX from 'xlsx';
+import { baseTimeOut } from '../configs/baseTimeOut';
+export const useProcessData = async (data, hierarchyType, type, tenantId, id, baseTimeOut, additionalDetails = {}) => {
+    let Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)